src/main/frontend/components/SiteShow.js

```diff
@@ -31,11 +35,38 @@ const SiteShow = (props) => {
       <div>
         <h1>Sorry this App/Dating Site can not be found</h1>
         <img src={"https://error404.fun/img/full-preview/1x/9.png"}
-             height="100%" alt="Page Not Found"/>
+          height="100%" alt="Page Not Found" />
       </div>
     )
   }
 
+  const postReviewForm = async (formPayload) => {
+    try {
+      const response = await fetch(
+        `/api/v1/reviews/${siteId}/addreview`, {
```

Comment:
To follow a more restful pattern we can have this send it's POST request to `/api/v1/sites/{siteId}/reviews`.  This would be the way to nest a resource.  Then on the backend there are a few ways to handle this, but the easiest way forward would be to add a POST endpoint for `/{siteId}/reviews` to the `SitesController`